1.0.3/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask==1.0.3
```

Comment:
I would rename this folder to be `1.0`, but specify the patch release in `requirements.txt`.